2018/03/19 16:17:12 [error] 15129#15129: *36 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request "site/error". in /var/www/proj1_MD_advanced/vendor/yiisoft/yii2/base/Module.php:537
Stack trace:
#0 /var/www/proj1_MD_advanced/vendor/yiisoft/yii2/web/ErrorHandler.php(108): yii\base\Module->runAction('site/error')
#1 /var/www/proj1_MD_advanced/vendor/yiisoft/yii2/base/ErrorHandler.php(111): yii\web\ErrorHandler->renderException(Object(yii\web\NotFoundHttpException))
#2 [internal function]: yii\base\ErrorHandler->handleException(Object(yii\web\NotFoundHttpException))
#3 {main}
Previous exception:
yii\web\NotFoundHttpException: Page not found. in /var/www/proj1_MD_advanced/vendor/yiisoft/yii2/web/Request.php:284
Stack trace:
#0 /var/www/proj1_MD_advanced/vendor/yiisoft/yii2/web/Application.php(82): yii\web\Request->resolve()
#1 /var/www/proj1_MD_advanced/vendor/yiisoft/yii2/base/Application.php(386): yii\web\Application->handleRequest(Object(yii\web\Request))
#2 /var/www/proj1_MD_advan" while reading upstream, client: 127.0.0.1, server: api.medic-a.local, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.2-fpm.sock:", host: "api.medic-a.local"
